no.po

```diff
@@ -0,0 +1,7947 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2021-07-26 12:08:06+01000\n"
+"PO-Revision-Date: 2022-02-22 13:12+0000\n"
+"Last-Translator: Sebastian Jentschke <sebastian.jentschke@uib.no>\n"
+"Language-Team: Norwegian Bokmål <https://hosted.weblate.org/projects/jamovi/"
+"jmv_i18n/nb_NO/>\n"
+"Language: nb_NO\n"
```

Comment:
don't forget this one here ... i'm wondering/hoping that `\n` will be ok.

Reply:
Can you figure it out (try whether they can be applied to jamovi). It is in the header so it actually shouldn't matter.
Otherwise, I have to find out whether there is a way to tweak that on Weblate but that might not be too easy.

Reply:
Do you mean the entry in the "Language:"-field? If so, can you change manually this time (or tell me to do it) and merge the pull request?
The problem is that Weblate is set up to pull from this repo (jamovi/jmv_i18n) so it may change the entries once the file name is updated in the repo.

Reply:
I just did the change.